### PR TITLE
fix headless

### PR DESCRIPTION
**Pull request by AllentDan**
Just want to visulize the saved picture.

### CODE REVIEW

File: tools/deploy.py [hunk 1]
```diff
@@ -247,37 +248,38 @@ def main():
...
-            kwargs=dict(
-                backend=Backend.PYTORCH,
-                output_file=osp.join(args.work_dir, 'output_pytorch.jpg'),
-                show_result=args.show),
-            ret_value=ret_value)
-    else:
-        logger.warning('\"visualize_model\" has been skipped may be because it\'s \
+    if is_display is None and args.show is True:
+        args.show = False
+        logger.warning('\"args.show\" has been forced to be False as it\'s \
```
Comment (grimoire) [inline tools/deploy.py:255]: You can use ` ''' "args.show" ... '''` to avoid escape character `\`.

**Review by grimoire — DISMISSED** [review]
LGTM

**Comment by RunningLeon** [comment]
@AllentDan Hi, maybe we should merge this PR on dev-v0.4.

**Comment by AllentDan** [comment]
> @AllentDan Hi, maybe we should merge this PR on dev-v0.4.

As we may merge dev-v0.4 into master soon while this PR is waiting for a user with Xavier to test, we may keep it to master.

**Comment by RunningLeon** [comment]
ok

**Comment by AllentDan** [comment]
Close as it does not work for Jeston nano. #238

**Pull request closed without merging**